src/context/CapturesContext.js

```diff
@@ -237,27 +244,210 @@ export function CapturesProvider(props) {
     setCaptures([]);
     setPage(0);
     setCaptureCount(null);
+    setInvalidateCaptureCount(true);
+  };
+
+  /**  VERIFICATION METHODS **/
+
+  const resetSelection = () => {
+    setCapturesSelected([]);
+    setCaptureImageAnchor(undefined);
+  };
+
+  const clickCapture = (payload) => {
+    const { captureId, isShift, isCmd, isCtrl } = payload;
+    if (!isShift && !isCmd && !isCtrl) {
+      setCapturesSelected([captureId]);
+      setCaptureImageAnchor(captureId);
+    } else if (isShift) {
+      log.debug('press shift, and there is an anchor:', captureImageAnchor);
+      //if no anchor, then, select from beginning
+      let indexAnchor = 0;
+      if (captureImageAnchor !== undefined) {
+        indexAnchor = captures.reduce((a, c, i) => {
+          if (c !== undefined && c.id === captureImageAnchor) {
```

Comment:
Same thing here, simplify to `if (c && c.id === captureImageAnchor)`

Reply:
I changed these to ternarys but can revert if it's not readable